<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LIN-2K3R9K3" failures="0" tests="8" name="testCaseRediffPortfolio.manageStocksTest" time="496.161" errors="1" timestamp="03 Dec 2024 08:39:42 GMT" skipped="0">
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="verifyTransactionHistory" time="62.625"/>
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="addStockTest" time="6.104"/>
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="verifyStockIsPresent" time="60.212"/>
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="verifyStockQuantity" time="60.210">
    <error message="Cannot invoke &quot;java.lang.Integer.intValue()&quot; because the return value of &quot;org.testng.ITestContext.getAttribute(String)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at testCaseRediffPortfolio.manageStocksTest.verifyStockQuantity(manageStocksTest.java:72)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifyStockQuantity -->
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="verifyTransactionHistory" time="62.484"/>
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="modifyStock" time="123.981"/>
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="verifyStockQuantity" time="60.213"/>
  <testcase classname="testCaseRediffPortfolio.manageStocksTest" name="verifyStockIsPresent" time="60.332"/>
</testsuite> <!-- testCaseRediffPortfolio.manageStocksTest -->
